python/lsst/ip/isr/crosstalk.py

```diff
@@ -334,11 +339,11 @@ def fromTable(cls, tableList):
         inDict['nAmp'] = metadata['NAMP']
 
         inDict['coeffs'] = coeffTable['CT_COEFFS']
-        if 'CT_ERRORS' in coeffTable:
+        if 'CT_ERRORS' in coeffTable.columns:
             inDict['coeffErr'] = coeffTable['CT_ERRORS']
-        if 'CT_COUNTS' in coeffTable:
+        if 'CT_COUNTS' in coeffTable.columns:
             inDict['coeffNum'] = coeffTable['CT_COUNTS']
-        if 'CT_VALID' in coeffTable:
+        if 'CT_VALID' in coeffTable.columns:
```

Comment:
Why now do you have to check `coeffTable.columns` instead of just `coeffTable`? Is this the only place where this needs to be changed?

Reply:
Without the change, astropy throws a
`astropy/table/row.py:76: FutureWarning: elementwise == comparison failed and returning scalar instead; this will raise an error or perform elementwise comparison in the future.
  return self.as_void() == other` warning that I found annoying.  I don't know why checking inclusion requires an `__eq__` run on the row, and didn't care to dig deeper.  Checking the `.columns` attribute should have the same behavior, and as it isn't calling whatever the other style is calling, it probably runs infinitesimally faster.